torch/_inductor/template_heuristics/triton.py

```diff
@@ -947,6 +948,15 @@ def __init__(self) -> None:
             (torch.float16, 256): FlexConfig(32, 64, 3, 4),
         }
 
+        self.flex_attn_bwd_autotune_configs_sm_100: list[FlexBwDConfig] = [
+            # See Note: flex bwd configs
+            FlexBwDConfig(BLOCK_M, BLOCK_N, BLOCK_N, BLOCK_M, s, 4)
+            for BLOCK_M in [32, 64]
+            for BLOCK_N in [32, 64]
```

Comment:
So basically you are excluding 128 for blackwell. How did you figure this out? Was it just trial-error or is this a known triton limitation for blackwell? Wondering if we need to propagate this finding all the way down to the triton developers.

Reply:
actually yeah that is a little weird can we maybe have them in here and add A TODO skip because of X and open an issue

Reply:
Yes, some for the benchmarks do fail with ULF, while some fail with OOM, and the rest (about half of them) are successful.